packages/ibm-products/src/components/CoachmarkStack/CoachmarkStack.tsx

```diff
@@ -36,6 +38,68 @@ import { CoachmarkTagline } from '../Coachmark/CoachmarkTagline';
 import { CoachmarkContext } from '../Coachmark/utils/context';
 import { COACHMARK_OVERLAY_KIND } from '../Coachmark/utils/enums';
 
+type Media =
+  | {
+      render?: () => ReactNode;
+    }
+  | {
+      filePaths?: string[];
+    };
+interface CoachmarkStackProps {
+  /**
+   * CoachmarkStack should use a single CoachmarkOverlayElements component as a child.
+   */
+  children: React.ReactNode;
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className?: string;
+  /**
+   * The label for the button that will close the Stack
+   */
+  closeButtonLabel?: string;
+  // Pass through to CoachmarkStackHome
+  /**
+   * The description of the Coachmark.
+   */
+  description: React.ReactNode;
+  /**
+   * The object describing an image in one of two shapes.
+   *
+   * If a single media element is required, use `{render}`.
+   *
+   * If a stepped animation is required, use `{filePaths}`.
+   *
+   * @see {@link MEDIA_PROP_TYPE}.
+   */
+  media: Media;
```

Comment:
As per PropType `media` prop is optional.

Reply:
should be updated now!